README.md

```diff
@@ -124,39 +124,74 @@ a proxy tool such as [mitmproxy](https://mitmproxy.org), or even just
 a script.
 
 A particularly simple way to proxy requests to a private bitcoind node
-is to make use of the [`corsproxy`](https://www.npmjs.com/package/corsproxy)
-npm module. Instructions to install and run the module are on its
-[home page](https://www.npmjs.com/package/corsproxy).
+is to make use of [`nginx`](https://nginx.org/). Instructions to install
+and run the program are on its [download page](https://nginx.org/en/download.html).
 
-Explicitly, install `corsproxy` with
+Explicitly, install `nginx` with
 
+```bash
+# MacOS
+brew install nginx
+
+# Debian Linux
+sudo apt install nginx
 ```
-$ npm install -g corsproxy
-```
 
-and then launch corsproxy
+copy the server conifiguration file to the appropriate location
```

Comment:
There hasn't been a mention of the config file yet so this could be confusing. Maybe mention the sample you've created and link to it as the one you'll want to copy to the nginx directory, and/or mention earlier that you'll need a server configuration file and that one is included in the project for reference.

Reply:
I cleared this up in the commit.